<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImageStampingConfigurationWidgetBase</class>
 <widget class="QWidget" name="ImageStampingConfigurationWidgetBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>737</width>
    <height>570</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="stampingSettingsGroupBox">
     <property name="title">
      <string>Styling Settings</string>
     </property>
     <layout class="QGridLayout" name="stampingSettingsLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="customFontStyleLabel">
        <property name="text">
         <string>Text style</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QgsFontButton" name="customFontStyleButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>1</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="customAlignmentLabel">
        <property name="text">
         <string>Horizontal alignment</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="customAlignmentComboBox"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="customImageDecorationLabel">
        <property name="text">
         <string>Image decoration</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QgsFileWidget" name="customImageDecorationFile" native="true"/>
      </item>
      <item row="3" column="0" colspan="2">
       <widget class="QCheckBox" name="forceStampingCheckBox">
        <property name="text">
         <string>Always stamp images taken using the QField camera</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="customDetailsGroupBox">
     <property name="title">
      <string>Details Template Settings</string>
     </property>
     <layout class="QVBoxLayout" name="previewLayout">
      <item>
       <widget class="QLabel" name="customDetailsLabel">
        <property name="text">
         <string>Details template</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPlainTextEdit" name="customDetailsTextEdit">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>120</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>300</height>
         </size>
        </property>
        <property name="placeholderText">
         <string>Leave empty to use the default details string</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="insertExpressionButton">
        <property name="text">
         <string>Insert/Edit Expression...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="previewLabel">
        <property name="wordWrap">
         <bool>true</bool>
        </property>
        <property name="text">
         <string></string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::MinimumExpanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsCollapsibleGroupBox</class>
   <extends>QGroupBox</extends>
   <header>qgscollapsiblegroupbox.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QgsDoubleSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header>qgsdoublespinbox.h</header>
  </customwidget>
  <customwidget>
   <class>QgsSpinBox</class>
   <extends>QSpinBox</extends>
   <header>qgsspinbox.h</header>
  </customwidget>
  <customwidget>
   <class>QgsExpressionBuilderWidget</class>
   <extends>QWidget</extends>
   <header>qgsexpressionbuilderwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QgsFileWidget</class>
   <extends>QWidget</extends>
   <header>qgsfilewidget.h</header>
  </customwidget>
  <customwidget>
   <class>QgsFontButton</class>
   <extends>QToolButton</extends>
   <header>qgsfontbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
